tests/modules/test_text_processor.py

```diff
@@ -236,6 +250,67 @@ def test_subscribe_twice_english(self, texting_mock, logging_info_mock, logging_
         expected_groups = ['Everyone - English', 'Text Sign Ups', 'Text Sign Ups - English']
         self.assertEqual(actual_groups, expected_groups)
 
+    @patch("logging.info")
+    @patch("modules.text_processor.Texter.send")
+    def test_pregnancy_birthdate_update_english(self, texting_mock, logging_info_mock):
+        t = TextProcessor(phone_number="1-111-1114")
+        self.assertFalse(Contact.objects.filter(name="Rose", phone_number="1-111-1114").exists())
+        first_response = t.process("JOIN ROSE 25-10-2012")
```

Comment:
So the flow for pregancy sign ups is that you text `JOIN` with a birth date in the future and then text `BORN` later with an updated DOB?